### DISCUSSION TITLE
Differenet Length of RFID Numbers for same Card

### DISCUSSION THREAD

**Discussion by FaJukeBox** [MiczFlor/RPi-Jukebox-RFID#1881] [Q&A]
Hello 

Just Build a new Phoniebox, an synchronized the the New Box (2.3 - 24258901 - master) withe the old (1.1.9-rc7) one, so that the Cards work on both boxes. The Cards didn't work on the new Box. When i read in the Cards on the new one, the numbers of the Card has 2 digits more, so no chance that they will work. (The RFID reader in both boxes is the same RC522). 

Would be great when I Don't need to register all Cards again? Ideas?

Thanks

**Comment by ozonorg** [comment]
See https://stackoverflow.com/questions/27481383/rfid-one-tag-gives-two-different-ids-in-two-different-readers
There is a bunch of similar posts.

It does not really matter if the reader is the same as there are different methods of reading an ID, shifting bits around, etc. 
2.3 most likely uses a different library. 

You may be able to transcode the lookup table. Otherwise you would have to change the python scripts. If you are not familiar with that, re-registering may be less effort.
